### DISCUSSION TITLE
初始化数据库失败

### DISCUSSION THREAD

**Discussion by ximengyi** [flipped-aurora/gin-vue-admin#1103] [Q&A] [answered]
最新的master分支下载到本地初始化的时候就失败了，有人遇到过这个问题吗 ？
这是错误消息
/init/initdbDuplicated key 'alias-name' in struct config.SpecializedDBDuplicated key 'alias-name' in struct config.SpecializedDB，

**Comment by pnck** [comment, marked as answer]
应该已经去掉重复的key了

https://github.com/flipped-aurora/gin-vue-admin/commit/cfeb6e1276d35bb73e7146c7dbcaa0d43d5a9343
https://github.com/flipped-aurora/gin-vue-admin/commit/4726ba0c0a0cf58488bfc6cdd0b6a89bd683a72b

**Reply by ximengyi** [reply]
多谢提醒，我是在master分支运行了，我才发现现在github默认主分支是main分支，master分支落后好多了非常感谢~
